django/db/backends/base/features.py

```diff
@@ -100,6 +100,9 @@ class BaseDatabaseFeatures:
     # deferred
     can_defer_constraint_checks = False
 
+    # Does the backend supports index on `text` field
+    supports_index_on_textfield = True
```

Comment:
Please add `supports_index_on_text_field = False` to the Oracle database features.